### PR TITLE
updated cosmos spark configs

### PR DESCRIPTION
**Pull request by moderakh**
updated cosmos spark configs

### CODE REVIEW

File: sdk/cosmos/azure-cosmos-spark_3-1_2-12/Samples/Pyspark-Sample.ipynb [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "  \"spark.cosmos.accountKey\" : cosmosMasterKey,\n",
     "  \"spark.cosmos.database\" : cosmosDatabaseName,\n",
     "  \"spark.cosmos.container\" : cosmosContainerName,\n",
-    "  \"spark.cosmos.read.inferSchemaEnabled\" : \"true\"                          \n",
+    "  \"spark.cosmos.read.inferSchema.enabled\" : \"true\"                          \n",
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos-spark_3-1_2-12/Samples/Pyspark-Sample.ipynb:34]: NIT - remove unnecessary whitespaces

File: sdk/cosmos/azure-cosmos-spark_3-1_2-12/docs/configuration-reference.md [hunk 2]
```diff
@@ -20,16 +20,17 @@ Configuration Reference:
...
 | `spark.cosmos.applicationName`      | None    | Application name  |
-| `spark.cosmos.preferredRegionsList`      | None    | Preferred regions list to be used for a multi region Cosmos DB account. This is a comma separated value (e.g., `[eastus,westus]`) provided preferred regions will be used as hint. You should use a collocated spark cluster with your Cosmos DB account and pass the spark cluster region as preferred region. See list of azure regions [here](https://docs.microsoft.com/dotnet/api/microsoft.azure.documents.locationnames?view=azure-dotnet) |
+| `spark.cosmos.preferredRegionsList`      | None    | Preferred regions list to be used for a multi region Cosmos DB account. This is a comma separated value (e.g., `[eastus,westus]` or `eastus,westus`) provided preferred regions will be used as hint. You should use a collocated spark cluster with your Cosmos DB account and pass the spark cluster region as preferred region. See list of azure regions [here](https://docs.microsoft.com/dotnet/api/microsoft.azure.documents.locationnames?view=azure-dotnet) |
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos-spark_3-1_2-12/docs/configuration-reference.md:23]: Looking at https://msdata.visualstudio.com/CosmosDB/_git/CosmosDB?path=%2FProduct%2FMicrosoft.Azure.Documents%2FSharedFiles%2FLocationNames.cs&_a=contents&version=GBmaster I believe the sample values are incorrect? "East US, West US" I think would be the correct format - or do we expect different value sin .Net and Java?

**Review by FabianMeiswinkel — APPROVED** [review]
Looks good to me except for the seemingly incorrect region values.

File: sdk/cosmos/azure-cosmos-spark_3-1_2-12/docs/configuration-reference.md (hunk 2 above)
Comment (moderakh) [reply sdk/cosmos/azure-cosmos-spark_3-1_2-12/docs/configuration-reference.md:23]: all should work, the implementation does both lowercasing and also removes whitespaces. "East US", "EASTUS", "east us", "eastus" all work.

Comment (moderakh) [reply sdk/cosmos/azure-cosmos-spark_3-1_2-12/docs/configuration-reference.md]: done. I changed the doc to "East US, West US".

File: sdk/cosmos/azure-cosmos-spark_3-1_2-12/Samples/Pyspark-Sample.ipynb (hunk 1 above)
Comment (moderakh) [reply sdk/cosmos/azure-cosmos-spark_3-1_2-12/Samples/Pyspark-Sample.ipynb:34]: the whitespace is from jupyter notebook formatting by `nbstripout` (not introduced in this PR). we should keep the formatting by `nbstripout` tool.

**Pull request merged**